pkg/api/v1/post_routes.go

```diff
@@ -0,0 +1,77 @@
+package v1
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	capconfig "github.com/els0r/goProbe/cmd/goProbe/config"
+	"github.com/els0r/goProbe/cmd/goProbe/flags"
+	"github.com/els0r/goProbe/pkg/capture"
+	"github.com/els0r/status"
+	"github.com/go-chi/chi"
+)
+
+func (a *API) postRequestRoutes(r chi.Router) {
+	// list actions here
+	r.Post("/_reload", a.handleReload)
+}
+
+func (a *API) handleReload(w http.ResponseWriter, r *http.Request) {
+	pp := r.FormValue("pretty") == "1"
+
+	if pp {
+		status.SetOutput(w)
+		status.Line("Reloading configuration")
+	}
+
+	var writeoutsChan chan<- capture.Writeout = a.c.WriteoutHandler.WriteoutChan
+
+	capconfig.Mutex.Lock()
```

Comment:
Ah, right, that's what I meant further above, making the Mutex anonymous part of the cfg, you could just elegantly call capconfig.Lock() etc...